plugins/cost-insights/api-report.md

```diff
@@ -404,13 +406,18 @@ export const EngineerThreshold = 0.5;
 // @public @deprecated (undocumented)
 export type Entity = common.Entity;
 
+// @public
+export const EntityCostInsightsContent: () => JSX.Element;
+
 // @public (undocumented)
 export class ExampleCostInsightsClient implements CostInsightsApi {
   // (undocumented)
   getAlerts(group: string): Promise<Alert[]>;
   // (undocumented)
   getDailyMetricData(metric: string, intervals: string): Promise<MetricData>;
   // (undocumented)
+  getEntityDailyCost(entity: Entity_2, intervals: string): Promise<Cost>;
```

Comment:
Unless the entire entity is required I suggest that we update the argument to `entityRef: string` instead to make it a bit more in line with other APIs.

Reply:
Sure, done :)